.circleci/cimodel/data/pytorch_build_data.py

```diff
@@ -167,6 +167,7 @@ def child_constructor(self):
         next_nodes = {
             "asan": AsanConfigNode,
             "xla": XlaConfigNode,
+            "mlc": MLCConfigNode,
```

Comment:
NB this isn't actually doing anything right now, you still don't have an MLC CI job